sale_triple_discount/models/sale_order_line.py

```diff
@@ -58,7 +58,8 @@ def _multiplicative_discount(self):
         final_discount = 1
         for discount in discounts:
             final_discount *= discount
-        return 100 - final_discount * 100
+        result = 100 - final_discount * 100
+        return round(result, 3)
```

Comment:
@ajaniszewska-dev Why 3 (and not another number ...)?

Reply:
IMHO, you should (at least) use the configuration one:

![image](https://github.com/OCA/sale-workflow/assets/19529533/5c012625-3581-48bb-8fc0-b947b0996a8f)

Reply:
@rousseldenis thank you for remark, corrected code.